db/migrate/20220428233523_drop_settings.rb

```diff
@@ -0,0 +1,15 @@
+class DropSettings < ActiveRecord::Migration[5.2]
+  def up
+    if table_exists?(:settings)
+      drop_table(:settings) do |t|
```

Comment:
Oh but then it gets dropped... huh

Reply:
Yeah, this gem doesn't seem to be compatible with Rails 6, so I removed it in favor of settings in the parameter store. You can see those in .env.example and secrets.yml